### ISSUE TITLE
PDF top margin off on Mac

### ISSUE THREAD

**Issue by tresf** [qzind/qz-print#173]
Settings:

``` javascript
var config = qz.configs.create("PDF", {size: {width: 8.27, height: 11.69}, density: 300, units: 'in'});
```

Using CUPS PDF driver:
![image](https://cloud.githubusercontent.com/assets/6345473/13009630/7cdc9e98-d16b-11e5-80e7-fa009a00f16b.png)

Expected output: https://github.com/qzind/qz-print/issues/156#issuecomment-183349840

Related: #156
Currently, DPI must be forced due to: #155

**Comment by akberenz** [comment]
Can you provide logs?

**Comment by tresf** [comment]
> Can you provide logs?

I'll work on that now.  On a side-note, if we can make trace-logs configurable (rather than a compile option), that would make the log turn-around a bit quicker.

**Comment by tresf** [comment]
Logs:

**Edit:** Updated with the JSON message in case there's something funny going on there. :+1: 

```
[INFO] 2016-02-12 09:56:20,460 @ qz.common.TrayManager:?
    Allowed localhost to connect to QZ
[DEBUG] 2016-02-12 09:56:20,539 @ qz.ws.PrintSocketClient:?
    Message: {"call":"getVersion","promise":{},"timestamp":1455288980534,"uid":"dkks3c"}
[WARN] 2016-02-12 09:56:20,540 @ qz.ws.PrintSocketClient:?
    Bad signature on request
[DEBUG] 2016-02-12 09:56:28,675 @ qz.ws.PrintSocketClient:?
    Message: {"call":"print","promise":{},"params":{"printer":{"name":"PDF"},"options":{"colorType":"color","copies":1,"jobName":null,"density":300,"duplex":false,"interpolation":"bicubic","margins":0,"orientation":null,"paperThickness":null,"printerTray":null,"rotation":0,"scaleContent":true,"size":{"width":8.27,"height":11.69},"units":"in","altPrinting":false,"encoding":null,"endOfDoc":null,"perSpool":1},"data":[{"type":"pdf","data":"file:///Users/tres/Desktop/demo/assets/pdf_sample.pdf"}]},"timestamp":1455288988653,"uid":"pj4j9c"}
[WARN] 2016-02-12 09:56:28,676 @ qz.ws.PrintSocketClient:?
    Bad signature on request
[INFO] 2016-02-12 09:56:30,096 @ qz.common.TrayManager:?
    Allowed localhost to print to PDF
[DEBUG] 2016-02-12 09:56:30,099 @ qz.printer.PrintServiceMatcher:?
    Searching for PrintService matching PDF
[DEBUG] 2016-02-12 09:56:30,328 @ qz.printer.PrintServiceMatcher:?
    Found 1 printers
[DEBUG] 2016-02-12 09:56:30,503 @ qz.printer.PrintServiceMatcher:?
    Found match: PDFwriter
[DEBUG] 2016-02-12 09:56:30,578 @ qz.ws.PrintSocketClient:?
    Using qz.printer.action.PrintPDF to print
[DEBUG] 2016-02-12 09:56:30,858 @ qz.printer.action.PrintPDF:?
    Parsed 1 files for printing
[TRACE] 2016-02-12 09:56:30,934 @ qz.printer.action.PrintPixel:?
    DPI: 300.0
[TRACE] 2016-02-12 09:56:30,936 @ qz.printer.action.PrintPixel:?
    Drawable area: 0.0,0.0:595.44006,841.68
[TRACE] 2016-02-12 09:56:30,937 @ qz.printer.action.PrintPixel:?
    [color, iso-a4, 30000x30000 dphi]
[INFO] 2016-02-12 09:56:30,954 @ qz.printer.action.PrintPixel:?
    Starting printing (1 copies)
[INFO] 2016-02-12 09:56:44,911 @ qz.ws.PrintSocketClient:?
    Printing complete
```

**Comment by tresf** [comment]
**Edit:** Updated with the JSON message in case there's something funny going on there. :+1:

**Comment by tresf** [comment]
Closed via 60615fc, 3f2b495

![image](https://cloud.githubusercontent.com/assets/6345473/13158264/2cf12950-d65a-11e5-83ae-8d45d13202b2.png)
